.env.example

```diff
@@ -35,7 +35,10 @@ NEXT_PUBLIC_OIDC_SCOPE="openid"
 
 # The provider’s endpoint that will perform authentication and authorization.
 NEXT_PUBLIC_OIDC_AUTHORITY="http://localhost:8888/auth/realms/conext"
-
+# OIDC login tab title
```

Comment:
not sure if we want dutch text in this project, as far as i know we are not using dutch anywhere else and using env vars for translations is not really a good option :)

Reply:
following our discussion, changed this text (and other instances of dutch words)